google-cloud-datastore/src/main/java/com/google/cloud/datastore/Cursor.java

```diff
@@ -22,17 +22,18 @@
 import com.google.common.base.MoreObjects.ToStringHelper;
 import com.google.common.io.BaseEncoding;
 import com.google.protobuf.ByteString;
-import com.google.protobuf.InvalidProtocolBufferException;
+
```

Comment:
FYI, The google style guide has changed/relaxed regarding [imports](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing).